sdks/python/apache_beam/pvalue.py

```diff
@@ -329,8 +329,9 @@ def _from_runtime_iterable(it, options):
     elif len(head) == 1:
       return head[0]
     raise ValueError(
-        'PCollection with more than one element accessed as '
-        'a singleton view.')
+        'PCollection of length %d with more than one element accessed as a '
```

Comment:
<!--thread_id:cc_136163159_t; commit:306a1dc4a1e072a65c739e179b5e1017e2069709; resolved:1-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> Let's use "size" instead of "length" as PCollections are unordered.

<!--section:body-->
Done.